### PR TITLE
Fc changes

### PR DESCRIPTION
**Pull request by Albly**
Both networks:
- FC_regressor - trains real and imag parts separately 
- FC_cat_regressor - concatenates real and imag, then trains

Also, configurations for networks have been added: 
- Activation type  - can be chosen from torch.nn. 
- dropout  - whether to use dropout after each activation 
- use_batchnorm - whether to use batchnorm1D after each Linear layer

Created 2 test configs (but actually both networks has the same params):
- Notes for model's params can be found there

### CODE REVIEW

**Review by laralex — APPROVED** [review]
I would rather remove .vscode files from there
